build.gradle

```diff
@@ -188,13 +173,6 @@ shadowJar {
     finalizedBy currentJar
 }
 
-
-task cloudJar(type: com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar)  {
```

Comment:
Should we temporarily keep the "cloudJar" task around as a direct alias for the normal build, to avoid breaking downstream users' build processes? Or is it better to force awareness of this change by removing the old build target?

Reply:
I'm in favor of removing it so they know something happened.

Reply:
Especially since I don't want to make a dummy cloud.jar which will be identical to the normal picard jar, so their builds would break anyway.